### PR TITLE
Do not show logo on full width header if no logo specified

### PR DESCRIPTION
**Pull request by wbreeze**
Prevents simply showing alt text on masthead when there is no specified logo file.

### CODE REVIEW

File: _includes/_masthead.html [hunk 1]
```diff
@@ -49,13 +51,15 @@
...
 <div id="masthead">
-	<div class="row">
-		<div class="small-12 columns">
-			<a id="logo" href="{{ site.url }}" title="{{ site.title }} – {{ site.slogan }}">
-				<img src="{{ site.url }}{{ site.baseurl }}/assets/img/{{ site.logo }}" alt="{{ site.title }} – {{ site.slogan }}">
-			</a>
-		</div><!-- /.small-12.columns -->
-	</div><!-- /.row -->
+{% if site.logo %}
+  <div class="row">
```
Comment (TWiStErRob) [inline _includes/_masthead.html]: I think indentation should be consistent with the rest of the file/project.

File: _includes/_masthead.html [hunk 2]
```diff
@@ -1,13 +1,15 @@
 {% if page.header == NULL and page.header.image_fullwidth == NULL and page.header.pattern == NULL and page.header.background-color == NULL and page.header.title == NULL %}
 
 <div id="masthead-no-image-header">
+{% if site.logo %}
```
Comment (TWiStErRob) [inline _includes/_masthead.html]: Why not hide the whole header instead of putting an empty div there?

File: _includes/_masthead.html (hunk 1 above)
Comment (wbreeze) [reply _includes/_masthead.html]: As you like.  And just for fun, [hard tabs or spaces](https://en.wikipedia.org/wiki/Indentation_style#Tabs.2C_spaces.2C_and_size_of_indent)

**Review by wbreeze — COMMENTED** [review]
Comments addressed. Thank you for those!

File: _includes/_masthead.html (hunk 1 above)
Comment (TWiStErRob) [reply _includes/_masthead.html]: [I prefer tabs](https://youtu.be/SsoOG6ZeyUI) at home (personalized tab size by the code's reader, easier navigation in editor, distinguished indentation vs alignment whitespace), but don't mind any size spaces or tabs as long as it's consistent in the project/company.

File: _includes/_masthead.html [hunk 3]
```diff
@@ -49,13 +51,15 @@
 {% elsif page.header.image_fullwidth %}
 
 <div id="masthead">
+{% if site.logo %}
```
Comment (TWiStErRob) [inline _includes/_masthead.html:54]: Empty div here as well
(did you try if hiding these div messes up layout or not?)

**Comment by wbreeze** [comment]
This one is needed.  Without it, you don't get the image.

File: _includes/_masthead.html (hunk 3 above)
Comment (TWiStErRob) [reply _includes/_masthead.html:54]: > This one is needed. Without it, you don't get the image.

Which image? only the logo is rendered inside. Is it JavaScript?

**Comment by wbreeze** [comment]
Yes. See line 11 of `_includes/_footer_scripts.html`

File: _includes/_masthead.html (hunk 3 above)
Comment (wbreeze) [reply _includes/_masthead.html:54]: The `page.header.image_fullwidth`. Yes, JavaScript. See line 11 of `_includes/_footer_scripts.html`

Comment (TWiStErRob) [reply _includes/_masthead.html:54]: k, thanks

**Review by TWiStErRob — APPROVED** [review]

**Comment by TWiStErRob** [comment]
@Phlow if you merge this, I suggest using "Squash and Merge" feature of GitHub (little down arrow on green merge button).

**Comment by Phlow** [comment]
OK. Thanks for the help. I still am not used how to merge, commit, squash and so on :)

**Pull request merged**